generators/generator-constants.js

```diff
@@ -31,15 +31,16 @@ const DOCKER_ELASTICSEARCH = 'elasticsearch:2.4.1';
 const DOCKER_KAFKA = 'wurstmeister/kafka:0.10.1.1';
 const DOCKER_ZOOKEEPER = 'wurstmeister/zookeeper:3.4.6';
 const DOCKER_SONAR = 'sonarqube:6.2-alpine';
-const DOCKER_JHIPSTER_CONSOLE = 'jhipster/jhipster-console:v2.0.1';
-const DOCKER_JHIPSTER_ELASTICSEARCH = 'jhipster/jhipster-elasticsearch:v2.0.1';
-const DOCKER_JHIPSTER_LOGSTASH = 'jhipster/jhipster-logstash:v2.0.1';
-const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v2.0.1';
-const DOCKER_CONSUL = 'consul:0.7.2';
-const DOCKER_CONSUL_CONFIG_LOADER = 'jhipster/consul-config-loader:v0.2.1';
-const DOCKER_PROMETHEUS = 'prom/prometheus:v1.4.1';
-const DOCKER_PROMETHEUS_ALERTMANAGER = 'prom/alertmanager:v0.5.1';
-const DOCKER_GRAFANA = 'grafana/grafana:4.0.2';
+const DOCKER_JHIPSTER_CONSOLE = 'jhipster/jhipster-console:v2.1.0';
+const DOCKER_JHIPSTER_ELASTICSEARCH = 'jhipster/jhipster-elasticsearch:v2.1.0';
+const DOCKER_JHIPSTER_LOGSTASH = 'jhipster/jhipster-logstash:v2.1.0';
+const DOCKER_JHIPSTER_IMPORT_DASHBOARDS = 'pbesson/jhipster-import-dashboards:latest';
```

Comment:
Just done with my phone lol :D :D
Can you look at https://hub.docker.com/r/jhipster/jhipster-import-dashboards/tags/ ?